src/components/shared/Footer/Footer.tsx

```diff
@@ -99,15 +98,15 @@ const renderMobileFooter = (updatedAt: string) => (
     </footer>
 )
 
-const renderUpdateDate = (updatedAt: string) => {
-  return NumberDateToWordDate(updatedAt)
-}
+// const renderUpdateDate = (updatedAt: string) => {
```

Comment:
We should remove this comment too, it's always available in version control if we need it